index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a class="my-logo" href="index.html">My Logo</a>
+      <nav>
+        <a class="bars" href="index.html"><img src="img/Union.png" alt="bars"/></a>
+      </nav>   
+    </div>
+  </header>
+  <main>
+    <section class="main-section">
+      <div class="section-content">
+        <h1>Hey There.<br>I’m Ricardo.</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>I can help you build a product, feature<br>or website Look through some of my<br>work and experience! If you like what you<br>see and have a project you need coded,<br>don’t hesitate to contact me.</p>
+        <nav>
+          <ul class="social">
+            <li><a href="https://github.com/"><img src="img/Gith.png" alt="Github"/></a></li>
+            <li><a href="https://www.linkedin.com/home/?originalSubdomain=cl"><img src="img/Lnk.png" alt="Linkedin"/></a></li>
+            <li><a href="https://www.instagram.com/"><img src="img/Hand.png"></a></li>
```

Comment:
- [ ] Please try to make the design as close as possible, As suggested by your previous code reviewer you did good work but still your design still not looks like figma template you need to work on the width to make your website responsive 
- [ ] ![www](https://user-images.githubusercontent.com/92458236/235819195-78526187-8cb3-4c6a-a50b-264aa43bee10.jpg)